diff --git a/node_modules/react-quill/lib/component.js b/node_modules/react-quill/lib/component.js
index 9e57723..1552a4a 100644
--- a/node_modules/react-quill/lib/component.js
+++ b/node_modules/react-quill/lib/component.js
@@ -211,7 +211,7 @@ ReactQuill.prototype.getEditor = function getEditor() {
 	if (!this.quill) throw new Error('Accessing editor before it is mounted');
 	return this.quill;
 };
-
+/*
 ReactQuill.prototype.getEditorNode = function getEditorNode() {
 	warn(
 		'`getEditorNode()` is deprecated and will be removed in a future version.' +
@@ -219,7 +219,7 @@ ReactQuill.prototype.getEditorNode = function getEditorNode() {
 	);
 	return ReactDOM.findDOMNode(this);
 };
-
+*/
 ReactQuill.prototype.getEditingArea = function getEditingArea() {
 	warn(
 		'`getEditingArea()` is deprecated and will be removed in a future version.' +
@@ -227,7 +227,7 @@ ReactQuill.prototype.getEditingArea = function getEditingArea() {
 	);
-	return ReactDOM.findDOMNode(this).querySelector('.ql-editor');
+	return this.quill.root;
 };
 
 ReactQuill.prototype.getSelection = function getSelection() {
